ui/src/main/webapp/wizards/sources/index.js

```diff
@@ -23,15 +23,16 @@ const WizardView = ({ id, children, clearWizard }) => (
 
 const Wizard = connect(null, { clearWizard })(WizardView)
 
+const stageMapping = {
+  welcomeStage: WelcomeStage,
+  discoveryStage: DiscoveryStage,
+  sourceSelectionStage: SourceSelectionStage,
+  confirmationStage: ConfirmationStage,
+  completedStage: CompletedStage
+}
+
 let StageRouter = ({ stage, messages }) => {
-  const stageMapping = {
-    welcomeStage: <WelcomeStage />,
-    discoveryStage: <DiscoveryStage />,
-    sourceSelectionStage: <SourceSelectionStage />,
-    confirmationStage: <ConfirmationStage />,
-    completedStage: <CompletedStage />
-  }
-  return (stageMapping[stage])
+  return React.createElement(stageMapping[stage])
```

Comment:
Either way of creating the react element works but the reason for making this change wasn't for syntax reasons - it was to pull the stage map out of the function so it didn't have to recreate the jsx mappings each time.